bound/kt/src/main/kotlin/web5/sdk/crypto/Dsa.kt

```diff
@@ -0,0 +1,6 @@
+package web5.sdk.crypto
+
+enum class Dsa {
```

Comment:
i see so we have to keep this in sync manually instead of using the rust ones.

I wonder if there can be a dynamic way to do this, create new enum at runtime that matches the names.. idk, but this is fine

Reply:
> I wonder if there can be a dynamic way to do this

If we find ourselves writing code to codegen based on codegen'd code, we are lost in the sauce.